cle/backends/cgc/cgc.py

```diff
@@ -17,23 +15,17 @@ class CGC(ELF):
     """
     is_default = True # Tell CLE to automatically consider using the CGC backend
 
-    def __init__(self, binary, *args, **kwargs):
-        if hasattr(binary, 'seek'):
-            filename = None
-            stream = PatchedStream(binary, [(0, ELF_HEADER)])
-        else:
-            filename = binary
-            stream = PatchedStream(open(binary, 'rb'), [(0, ELF_HEADER)])
-        kwargs['filename'] = filename
-        super(CGC, self).__init__(stream, *args, **kwargs)
+    def __init__(self, binary, binary_stream, *args, **kwargs):
+        binary_stream = PatchedStream(binary_stream, [(0, ELF_HEADER)])
+        super(CGC, self).__init__(binary, binary_stream, *args, **kwargs)
```

Comment:
I think we can use the simpler style `super().__init__(...)` now.

Reply:
I have now fixed every single instance of this in CLE. I hope you're happy.